src/app/SharpRaven/Data/SentryStacktrace.cs

```diff
@@ -91,7 +84,7 @@ public override string ToString()
 
             StringBuilder sb = new StringBuilder();
 
-            foreach (var frame in Frames)
+            foreach (var frame in Frames.Reverse())
```

Comment:
Excellent! Think you can add a little comment above the two `Reverse()` statements to explain why they are performed?